libs/@blockprotocol/type-system/package.json

```diff
@@ -63,7 +63,6 @@
     "@rollup/plugin-typescript": "9.0.2",
     "@rollup/plugin-wasm": "6.0.1",
     "@types/jest": "29.2.5",
-    "@types/node": "18.11.10",
```

Comment:
What's the reason for this removal? Not required at all?

Reply:
It was removed as part of https://github.com/blockprotocol/blockprotocol/commit/3bea66e17d0cbf71c387a854f509d22d230681f9, I introduced it to fix linting under a newer version of TS which I accidentally added to the PR.